Developing-new-features-with-feature-gating.md

```diff
@@ -93,9 +199,37 @@ We use the following principles to determine the value of a feature flags:
 
 - Each feature flag has multiple *rules*, its value changes to the value specified in the first rule that is matched. If no rule is matched, its value remains default.
 - Each rule has multiple *filters*, and a rule is matched only when all of its filters are matched.
-- Each filter describes a scenario, there are multiple types of filters like `server mode`, `client type`, so that you can you can describe scenarios like `server mode = dev OR server mode = test`.
-- Each filter can have multiple conditions, and a filter is matched if any of the conditions are matched.
+- Each filter describes a scenario. There are multiple types of filters (like `server mode` and `client type`) which can be used in combination with each other to describe various scenarios (like `server mode = dev OR server mode = test`).
+- Each filter can have multiple conditions, and a filter is matched if any of the conditions specified in the rule are matched.
+
+For example, if we want a feature flag to be enabled only in dev environments, we can configure the flag's rules/filters as shown below.
```

Comment:
FWIW this example may be a bit confusing as it illustrates yet another way to gate on environment type. Perhaps giving an example of blocking for a specific browser might be a bit more clear?

Reply:
You're right, changed it to rules on browser types.